lib/supplejack_api/engine.rb

```diff
@@ -39,8 +39,8 @@ class Engine < ::Rails::Engine
   end
 
   def self.setup(&block)
-    config.record_class = Record
-    config.preview_record_class = PreviewRecord
+    # config.record_class = Record
```

Comment:
I guess this is not needed anymore?

Reply:
Yeah bro we got rid of it awhile ago, I think this part was missed out.